apps/site/lib/site_web/templates/schedule/_timetable.html.eex

```diff
@@ -13,9 +13,9 @@
       <div class="m-timetable__cell m-timetable__cell--gray m-timetable__cell--first-column m-timetable__cell--first-column-header m-timetable__row-header--empty"></div>
       <div class="m-timetable__col-headers" aria-hidden="true">
         <%= if should_show_scroll_controls? do %>
-          <span class="m-timetable__trains-label">Trains</span>
-          <button class="m-timetable__scroll-btn m-timetable__scroll-btn--left btn btn-outline-primary btn-sm" data-scroll="earlier"><%= fa "angle-left m-timetable__scroll-btn-arrow" %> Earlier Trains</button>
-          <button class="m-timetable__scroll-btn m-timetable__scroll-btn--right btn btn-outline-primary btn-sm" data-scroll="later">Later Trains <%= fa "angle-right m-timetable__scroll-btn-arrow" %></button>
+          <span class="m-timetable__trains-label"><%= trains_or_boats(@route.type) <> "s" %></span>
```

Comment:
I think you can replace `trains_or_boats(@route.type)` with `Routes.Route.vehicle_name(@route)`! 😃

Reply:
OOH I just realized `Routes.Route.vehicle_name` is only used in two places right now and they're both to show realtime data, so they don't even show ferries yet. @hannahpurcell I give you permission to edit `Routes.Route.vehicle_name` so that it outputs "Boat" so that you can use it here.